### DISCUSSION TITLE
I don't have Internet access but I do have a network

### DISCUSSION THREAD

**Discussion by Segu1212** [devmfc/debian-on-amlogic#8] [Q&A]
Hello, great job, it works for my box but I have a problem, I don't have Internet, I think the DNS doesn't work or I don't know what's happening, I also tried your other previous versions and it was the same result, I clarify that if it has an IP when I connect to the network I can even log in through ssh but I don't have Internet I do apt update and there is no reaction![IMG_20231003_195614.jpg](https://github.com/devmfc/debian-on-amlogic/assets/146899206/715da568-af52-42ef-9d19-988316183ca3)

**Comment by devmfc** [comment]
Sorry, my bad...
Execute `ln -sf ../run/systemd/resolve/stub-resolv.conf /etc/resolv.conf` and it will be resolved I think.

Or edit  /etc/resolv.conf and edit ypur DNS server there.

**Comment by Segu1212** [comment]
Thank you.  The problem has now been resolved.
 Good job, so far only the last update you uploaded has worked for me.
  I have some details about the Ubuntu version.  Yes, desktop mode works, but in Debian desktop mode does not work.  In the last update you uploaded, I couldn't apply the sh script.

  I also have problems with the Driver, everything works but the most important detail is that I do not have audio, my box has an Android s905 x2 processor.  I tried to try a dual USB Bluetooth with wi-fi rtl2723bu but it doesn't work because there is no driver.  I also tried a generic USB audio card but it doesn't work, it doesn't recognize it.

**Comment by devmfc** [comment]
Have you tried audio via HDMI? For the S905X2 it should work...

I create these images purely as servers. Just for fun I occasionally try desktop on Ubuntu. But I think these boxes are really to slow for real desktop usage without all the vendor kernel video/graphics acceleration stuff. More and more is coming to mainline and it gets a bit better every release, but I wouldn't recommend it ;-)

The wifi driver foor rtl8723bu should be there (rtl8xxxu.ko).  Maybe you're missing some firmware. Try `apt-get install  linux-firmware`.
The kernel headers are in the release files. So it should be possible to compile your own wifi drivers. I can't help you with that, sorry...

**Reply by Segu1212** [reply]
Thanks I was able to get rtl8723bu working but I don't have HDMI audio but I finally have Bluetooth audio

**Reply by devmfc** [reply]
Great to hear! Did you compile the driver or was the firmware enough?
